recipes/recipes_emscripten/llvm/recipe.yaml

```diff
@@ -12,9 +12,12 @@ source:
   patches:
   - patches/shift_temporary_files_to_tmp_dir.patch
   - patches/enable_exception_handling.patch
+  - patches/fix_initialization_logs.patch
+  - patches/fix_transitive_headers.patch
```

Comment:
The patch fixing including transitive headers will be removed by the next release 20.1.7 

Should help us with catching errors instead of hitting the assert block as seen

![image](https://github.com/user-attachments/assets/54f00e35-4c2d-4cbd-9716-3256b2dbda56)